packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -124,3 +124,57 @@ const myStack = new Stack(app, 'MyStack');
 declare const application: appreg.Application;
 application.associateStack(myStack);
 ```
+
+## Sharing
+
+You can share your AppRegistry applications and attribute groups with AWS Organizations, Organizational Units (OUs), AWS accounts within an organization, as well as IAM roles and users. AppRegistry requires that AWS Organizations is enabled in an account before deploying a share of an application or attribute group.
+
+### Sharing an application
+
+```ts
+import * as iam from '@aws-cdk/aws-iam';
+declare const application: appreg.Application;
+declare const myRole: iam.IRole;
+declare const myUser: iam.IUser;
+application.shareResource({
```

Comment:
I don't like the word `Resource` here, a little abstract. Why not `application.share()`, or `application.shareApplication()` if you want a noun?

Reply:
I opted for the noun approach for this function on both applications and attribute groups.